index.js

```diff
@@ -2,13 +2,22 @@ const express = require('express');
 const helmet = require('helmet');
 
 const server = express();
+const router = require('./zoos/router.js');
```

Comment:
to maintain consistency, I'd recommend this be `zoos`, but obviously will still work as is